apps/native-component-list/src/screens/AV/AudioPlayer.tsx

```diff
@@ -22,6 +22,9 @@ interface Props {
   source: PlaybackSource;
 }
 
+// TODO: Grab the type from expo-av.
```

Comment:
How can I do this? I've declared the type in `AV.ts` but I can't access it from `expo-av`.

Reply:
I see that there's a problem with exporting the types from this module.
As you can see in [`package.json`](https://github.com/expo/expo/blob/master/packages/expo-av/package.json) the entry point for the types for this module is [`build/index.d.ts`](https://github.com/expo/expo/blob/master/packages/expo-av/build/index.d.ts) that comes from [`src/index.ts`](https://github.com/expo/expo/blob/master/packages/expo-av/src/index.ts). This latter file does not re-export the exported values nor types from [`src/AV.ts`](https://github.com/expo/expo/blob/master/packages/expo-av/src/AV.ts) directly. Unfortunately, I don't know why it's written that way (😅), but I believe it should be fixed to export all the that are exposed in the module's API 💪  
I suggest adding the following line to the [`src/index.ts`](https://github.com/expo/expo/blob/master/packages/expo-av/src/index.ts) file, rebuild the package and you'll be able to use declared in [`src/AV.ts`](https://github.com/expo/expo/blob/master/packages/expo-av/src/AV.ts) type 🥳 
```diff
  import * as Audio from './Audio';
  export { Audio };
  export {
    default as Video,
    VideoProps,
    VideoState,
    AVPlaybackNativeSource,
    AVPlaybackStatus,
    AVPlaybackStatusToSet,
    VideoNativeProps,
    VideoNaturalSize,
    VideoReadyForDisplayEvent,
    ResizeMode,
    VideoFullscreenUpdateEvent,
  } from './Video';
+ export { AVPlaybackMetadata } from './AV';
```